whist/server/services/game_db_service.py

```diff
@@ -52,7 +52,7 @@ def get_by_name(cls, game_name: str) -> GameInDb:
         :param game_name: of the game
         :return: the game database object
         """
-        game = cls._games.find_one({'game_name': game_name})
+        game = cls._games.find_one({'table.name': game_name})
```

Comment:
is this correct?

Reply:
Yes because the name is in the Table attribute.

https://github.com/Whist-Team/Whist-Server/blob/4a40b4a34aae6a2d1618c5aee921dab474acfbdb/whist/server/database/game.py#L21